documentation/learningPath/testing.md

```diff
@@ -2,3 +2,86 @@
 ### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2FlearningPath%2Ftesting)
 
 # Testing
+
+## Running Unit Tests
+
+Tests can be executed with the command line (via [build.cmd](../../Build.cmd) -test), as part of the PR build, or in Visual Studio. Note that because of limited resources in the build pool, tests ran from the command line or in the build pool are serialized. This avoids test failures associated with parallel testing. Visual Studio does not have such restrictions and is best used for individual tests and test investigations. 
+
+The framework of the test assemblies is controlled by [TestTargetFrameworks](../../eng/Versions.props). The test itself is attributed with a particular framework based on the [TargetFrameworkMonikerTraitAttribute](../../src/tests/Microsoft.Diagnostics.Monitoring.TestCommon/TargetFrameworkMonikerTraitAttribute.cs).
+
+## Unit Tests
+
+- [Microsoft.Diagnostics.Monitoring.Tool.UnitTests](../../src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests)
+- [Microsoft.Diagnostics.Monitoring.WebApi.UnitTests](../../src/tests/Microsoft.Diagnostics.Monitoring.WebApi.UnitTests/)
+- [CollectionRuleActions.UnitTests](../../src/Tests/CollectionRuleActions.UnitTests/)
+
+Unit test assemblies directly reference types from various dotnet-monitor assemblies. However, since most of dotnet-monitor heavily relies on code injection, there are utility classes to simplify unit test creation. 
+
+- [TestHostHelper](../../src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTestCommon/TestHostHelper.cs) can be used to setup a basic unit test scenario using dependency injection.
+- [CollectionRuleOptionsExtensions](../../src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTestCommon/Options/CollectionRuleOptionsExtensions.cs) can be used to easily create collection rules from configuration.
+
+## Functional Tests
+
+- [Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests](../../src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests)
+- [Microsoft.Diagnostics.Monitoring.UnitTestApp](../../src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/)
+
+Functional tests are composed of 3 main parts:
+- The test itself, which sets up and validates the results.
+- An instance of dotnet-monitor
+- An instance of an application that is being monitored (from the UnitTestApp assembly)
+
```

Comment:
This section looks a bit odd from a formatting perspective as it is right now (there's an almost imperceptible gap between the "3 main parts" and these other bullet points). Consider adjusting how this is displayed.